src/event/player/PlayerBucketEvent.php

```diff
@@ -62,7 +64,7 @@ public function __construct(Player $who, Block $blockClicked, int $blockFace, It
 	/**
 	 * Returns the bucket used in this event
 	 *
-	 * @return Item
+	 * @return LiquidBucket|Bucket|Item
```

Comment:
This is orthogonal to adding liquid type API, and also unnecessary since `Item` includes these types.

Reply:
Yes, but how IDE understand that Item has getLiquid method?

Reply:
`instanceof`, same as anything.

Reply:
Well, method called "getBucket" and should already it, not any item